src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -95,7 +95,6 @@ internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider
 
             try
             {
-                Debug.Assert(PathUtilities.IsAbsolute(keyFilePath));
```

Comment:
:+1: I have a bug on me to remove it :)

https://github.com/dotnet/roslyn/issues/8360